src/DssSpell.sol

```diff
@@ -55,139 +47,39 @@ contract DssSpellAction is DssAction {
+        // ---------- Spark - AAVE Revenue Share Payment ----------
+        // Forum: https://forum.makerdao.com/t/spark-aave-revenue-share-calculation-payment-1-q3-2023/22486
+        // MIP: https://mips.makerdao.com/mips/details/MIP106#9-4-1-spark-protocol-aave-revenue-share
 
-        // NOTE: Update collateral price to propagate the changes
-        DssExecLib.updateCollateralPrice("RWA007-A");
+        // Send 2889 DAI from Surplus Buffer to 0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c
+        // Skip on goerli
 
-        // ---------- Set up Governance Facilitator Streams  ----------
-        // Forum: https://forum.makerdao.com/t/mip102c2-sp16-mip-amendment-subproposal/21579
-        // Poll: https://vote.makerdao.com/polling/QmSovaxn
-        // NOTE: Skip on goerli
+        // ---------- Immunefi CU MKR Vesting Transfer ----------
+        // Immunefi CU - 6.34 MKR - 0xd1F2eEf8576736C1EbA36920B957cd2aF07280F4
+        // Forum: https://forum.makerdao.com/t/mip39c3-sp13-removing-is-001/22392
+        // MIP: https://mips.makerdao.com/mips/details/MIP40c3SP41#sentence-summary
+        // Skip on goerli
 
-        // JanSky | 2023-10-01 00:00:00 to 2024-09-30 23:59:59 | 504,000.00 DAI | 0xf3F868534FAD48EF5a228Fe78669cf242745a755
-        // VoteWizard | 2023-10-01 00:00:00 to 2024-09-30 23:59:59 | 504,000.00 DAI | 0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1
-        // JanSky | 2023-10-01 00:00:00 to 2024-09-30 23:59:59 | 216.00 MKR | 0xf3F868534FAD48EF5a228Fe78669cf242745a755
-        // VoteWizard | 2023-10-01 00:00:00 to 2024-09-30 23:59:59 | 216.00 MKR | 0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1
+        // ---------- Housekeeping - GUSD & USDP - Add Jar & Conduit Contracts to Chainlog ----------
+        // Forum: https://forum.makerdao.com/t/proposed-housekeeping-items-upcoming-executive-spell-2023-11-01/22477
 
+        // Add `RwaJar` at 0xf2E7a5B83525c3017383dEEd19Bb05Fe34a62C27 as MCD_PSM_GUSD_A_JAR
+        // Skip on goerli
 
-        // ---------- BA Labs MKR Distribution  ----------
-        // Forum: https://forum.makerdao.com/t/mip40c3-sp25-risk-core-unit-mkr-compensation-risk-001/9788
-        // Poll: https://vote.makerdao.com/polling/QmUAXKm4
-        // NOTE: Skip on goerli
+        // Add `RwaSwapInputOutputConduit2` at 0x6934218d8B3E9ffCABEE8cd80F4c1C4167Afa638 as MCD_PSM_GUSD_A_INPUT_CONDUIT_JAR
+        // Skip on goerli
 
-        // BA Labs - 175 MKR - 0x5d67d5B1fC7EF4bfF31967bE2D2d7b9323c1521c
+        // Add `RwaJar` at 0x8bF8b5C58bb57Ee9C97D0FEA773eeE042B10a787 as MCD_PSM_PAX_A_JAR
+        // Skip on goerli
 
-        // ---------- AVC Member Compensation  ----------
-        // Forum: https://forum.makerdao.com/t/avc-member-participation-rewards-q3-2023/22349
-        // Poll: https://vote.makerdao.com/polling/QmSovaxn#poll-detail
-        // NOTE: Skip on goerli
+        // Add `RwaSwapInputConduit2` at 0xDa276Ab5F1505965e0B6cD1B6da2A18CcBB29515 as MCD_PSM_PAX_A_INPUT_CONDUIT_JAR
+        // Skip on goerli
```

Comment:
[Another instruction](https://docs.google.com/spreadsheets/d/1w_z5WpqxzwreCcaveB2Ye1PP5B8QAHDglzyxKHG3CHw/edit#gid=1582521921&range=B48) is missing here:
```diff
+
+        // Call `<conduit>.rely(MCD_ESM)` to allow ESM module to `deny` the pause proxy in SwapInputConduit contracts
+        // Skip on goerli
```